tests/UtilsTests/EventTagUtilsTest.php

```diff
@@ -39,12 +46,60 @@ public function testGetRevenueValueWithInvalidRevenueTag() {
         $this->assertNull(EventTagUtils::getRevenueValue(array('revenue' => '65536')));
         $this->assertNull(EventTagUtils::getRevenueValue(array('revenue' => true)));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('revenue' => false)));
         $this->assertNull(EventTagUtils::getRevenueValue(array('revenue' => array(1, 2, 3))));
     }
 
     public function testGetRevenueValueWithRevenueTag() {
-        $this->assertEquals(65536, EventTagUtils::getRevenueValue(array('revenue' => 65536)));
-        $this->assertEquals(9223372036854775807, EventTagUtils::getRevenueValue(array('revenue' => 9223372036854775807)));
-        $this->assertEquals(0, EventTagUtils::getRevenueValue(array('revenue' => 0)));
+        $this->assertSame(65536, EventTagUtils::getRevenueValue(array('revenue' => 65536)));
+        $this->assertSame(9223372036854775807, EventTagUtils::getRevenueValue(array('revenue' => 9223372036854775807)));
+        $this->assertSame(0, EventTagUtils::getRevenueValue(array('revenue' => 0)));
+    }
+
+    public function testGetNumericValueInvalidArgs() {
+       $this->assertNull(EventTagUtils::getNumericValue(null));
+       $this->assertNull(EventTagUtils::getNumericValue(0.5));
+       $this->assertNull(EventTagUtils::getNumericValue(65536));
+       $this->assertNull(EventTagUtils::getNumericValue(9223372036854775807));
+       $this->assertNull(EventTagUtils::getNumericValue('65536'));
+       $this->assertNull(EventTagUtils::getNumericValue(true));
+       $this->assertNull(EventTagUtils::getNumericValue(false));
+   }
+
+   public function testGetNumericValueNoValueTag() {
+        $this->assertNull(EventTagUtils::getNumericValue(array()));
+        $this->assertNull(EventTagUtils::getNumericValue(array('non-value' => 42)));
+    }
+
+    public function testGetNumericValueWithInvalidValueTag() {
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => null)));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => 'abcd')));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => true)));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => false)));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => array(1, 2, 3))));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => array())));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => '1,234')));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => floatval(NAN))));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => (self::max_float*10))));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => floatval(INF))));
+        $this->assertNull(EventTagUtils::getRevenueValue(array('value' => floatval(-INF))));
     }
+
+    // Test that the correct numeric value is returned
+    public function testGetNumericValueWithValueTag() {
```

Comment:
There are some test cases missing, please check python code. 
https://github.com/optimizely/python-sdk/blob/master/tests/helpers_tests/test_event_tag_utils.py#L82
Line 101 to 131 test cases are missing.

Reply:
I have added all of the testcases. The invalid value test cases have been added above in their relevant methods